2025-09-29 16:28:38,178 [INFO ] => loading model checkpoint
/Users/leo/Desktop/AML/Multi-GNN/inference.py:71: FutureWarning: You are using `torch.load` with `weights_only=False` (the current default value), which uses the default pickle module implicitly. It is possible to construct malicious pickle data which will execute arbitrary code during unpickling (See https://github.com/pytorch/pytorch/blob/main/SECURITY.md#untrusted-models for more details). In a future release, the default value for `weights_only` will be flipped to `True`. This limits the functions that could be executed during unpickling. Arbitrary objects will no longer be allowed to be loaded via this mode unless they are explicitly allowlisted by the user via `torch.serialization.add_safe_globals`. We recommend you start setting `weights_only=True` for any use case where you don't have full control of the loaded file. Please open an issue on GitHub for any issues related to this experimental feature.
  checkpoint = torch.load(f'{data_config["paths"]["model_to_load"]}/checkpoint_{args.unique_name}.tar')
Traceback (most recent call last):
  File "/Users/leo/Desktop/AML/Multi-GNN/main.py", line 41, in <module>
    main()
  File "/Users/leo/Desktop/AML/Multi-GNN/main.py", line 38, in main
    infer_gnn(tr_data, val_data, te_data, tr_inds, val_inds, te_inds, args, data_config)
  File "/Users/leo/Desktop/AML/Multi-GNN/inference.py", line 71, in infer_gnn
    checkpoint = torch.load(f'{data_config["paths"]["model_to_load"]}/checkpoint_{args.unique_name}.tar')
  File "/Users/leo/miniconda3/envs/multignn/lib/python3.9/site-packages/torch/serialization.py", line 1065, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/Users/leo/miniconda3/envs/multignn/lib/python3.9/site-packages/torch/serialization.py", line 468, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/Users/leo/miniconda3/envs/multignn/lib/python3.9/site-packages/torch/serialization.py", line 449, in __init__
    super().__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: '/Users/leo/Desktop/AML/Multi-GNN/models (e.g for inference or fine-tuning)/checkpoint_.tar'
